pep-0001.txt

```diff
@@ -229,45 +255,69 @@ PEP Review & Resolution
-frequently with PEPs where the BDFL has granted in principle approval for
-*something* to be done, but there are details that need to be worked out
-before the PEP can be accepted.
-
-If the final decision on a PEP is to be made by a delegate rather than
-directly by the BDFL, this will be recorded by including the
-"BDFL-Delegate" header in the PEP.
-
-PEP review and resolution may also occur on a list other than python-dev
-(for example, distutils-sig for packaging related PEPs that don't
-immediately affect the standard library). In this case, the "Discussions-To"
-heading in the PEP will identify the appropriate alternative list where
-discussion, review and pronouncement on the PEP will occur.
+the BDFL-Delegate for that PEP, and they will then have the authority to approve
+(or reject) that PEP.
+
+The designated decision maker for each PEP is recorded in the "BDFL-Delegate"
+header in the PEP.
+
+Such self-nominations are accepted by default, but may be explicitly declined by
+the Steering Council. Possible reasons for the Steering Council declining a
+self-nomination as BDFL-Delegate include, but are not limited to, perceptions of
+a potential conflict of interest (e.g. working for the same organisation as the
+PEP submitter), or simply considering another potential BDFL-Delegate to be
+more appropriate. If core developers (or other community members) have concerns
+regarding the suitability of a BDFL-Delegate for any given PEP, they may ask
+the Steering Council to review the delegation.
+
+If no volunteer steps forward, then the Steering Council will approach core
+developers (and potentially other Python community members) with relevant
+expertise, in an attempt to identify a candidate that is willing to serve as
+BDFL-Delegate for that PEP. If no suitable candidate can be found, then the
+PEP will be marked as Deferred until one is available.
+
+While BDFL-Delegates hold final authority over acceptance or rejection of a PEP,
+they are expected to take the advice and perspectives of other core developers
+into account, and are free to seek additional guidance from the Steering Council
+at any time. BDFL-Delegates may also choose to step down, or be asked to step
+down by the Council, in which case a new BDFL-Delegate will be appointed in the
+same manner as for a new PEP (including deferral of the PEP if no suitable
+replacement can be found).
+
+With the approval of the Steering Council, PEP review and resolution may also
+occur on a list other than python-dev (for example, distutils-sig for packaging
+related PEPs that don't immediately affect the standard library). In these
+cases, the "Discussions-To" heading in the PEP will identify the appropriate
+alternative list where discussion, review and pronouncement on the PEP will
+occur.
+
+When such standing delegations are put in place, the Steering Council will
+maintain sufficient records to allow subsequent Councils to understand the
```

Comment:
Maybe the core devs and the general public should also be audiences of these records?

Reply:
Yes!

Reply:
Revised wording makes it explicit that these records should be public and for a general audience: https://github.com/python/peps/pull/896/commits/a50dbc7e1e69341645d27851c8a202f132fff05d

(I hadn't even noticed that the original wording technically allowed those records to be "for Steering Council eyes only")